### ISSUE TITLE
Package manager now uses a <Version> child element?

### ISSUE THREAD

**Issue by tonyferrell** [dotnet/project-system#2947]
I'm not sure if we did something wrong, but after upgrading to the latest version of VS, our packages are included a new (much harder to read) format.

For instance, they used to be formatted like 
```
<PackageReference Include="Common.Logging" Version="3.4.1" />
```
But when they're touched by the package manager, they are now of the format 
```
<PackageReference Include="Common.Logging">
  <Version>3.4.1</Version>
</PackageReference>
```
Are we doing something weird? Is this expected? I can't find any documentation on it being changed (other than some old references that don't look relevant)

**Comment by davkean** [comment]
This is known and fixed in recent internal builds. @jviau Was this fixed in 15.5?

**Comment by natidea** [comment]
Yes this is fixed in the next Preview. Dupe of [507675](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/507675)
